vignettes/ReactomeContentService4R.Rmd

```diff
@@ -231,19 +231,62 @@ getOrthology("R-SSC-69541", species = "human")
 
 The Reactome [diagram exporter](https://reactome.org/dev/content-service/diagram-exporter) has been mentioned a little bit in 'Instances in Reactions' section, here we can reconstruct a few more examples.
 
-Export animaged gifs of Enhanced High Level Diagrams ([**EHLDs**](https://reactome.org/icon-info/ehld-specs-guideline)) of Pathway "Cell Cycle, Mitotic" (stId R-HSA-69278) with overlaid analysis results, and highlight "S Phase" (stId R-HSA-69242):
+What's more, Reactome offers a pathway [Analysis Service](https://reactome.org/dev/analysis) that supports enrichment and expression analysis. The diagram exporter allows you to overlay the results of the analysis on top of the exported diagrams. To do so, use the token argument to specify the unique token associated with the performed analysis.
 
-```{r gif}
-gif <- exportImage(id = "R-HSA-69278", output = "diagram", format = "gif",
-                   sel = "R-HSA-69242", token = "MjAyMDA2MTcyMDM5NDBfMzU2")
-print(gif)
+The [ReactomeGSA](https://github.com/reactome/ReactomeGSA) package is an R client to the web-based Reactome Analysis Service, so we could perform analysis in R then access the token.
+
+```{r ReactomeGSA, eval=FALSE}
+library(ReactomeGSA)
+library(ReactomeGSA.data)
```

Comment:
'mgcv' gave compilation issues. The solution is to download it manually and not install sources from package that needs compilation:

`> install.packages("mgcv")`
`Do you want to install from sources the package which needs compilation? (Yes/no/cancel) no`

This allowed me to get past the 'my_request' examples, but still fails at the 'result' part